config/env/default.js

```diff
@@ -28,5 +28,5 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: './modules/core/client/img/brand/favicon.ico'
```

Comment:
@codydaig @rodp82 I don't see where else this is used. Have you seen it used somewhere else in the application? 

As for the question of whether we need `./`, the answer is no. We don't need it. The `favicon` library doesn't require it, so we're probably better off not changing this in case others have used this setting in other areas of the the application.

After testing, I am questioning why we're using `path.resolve` here https://github.com/meanjs/mean/pull/1014/files#diff-16e0a4c137ae50cf68b1d2c3c914be85R40

This will set the URL to an absolute path, which would expose the directory structure of where this file is located. We probably don't want that, and resolving the path isn't necessary for these purposes.

IMO, this is the only change that is required in this PR https://github.com/meanjs/mean/pull/1014/files#diff-16e0a4c137ae50cf68b1d2c3c914be85R69

I have tested the favicon, with just this change. Everything is working as expected.